http/graphql/pom.xml

```diff
@@ -15,50 +15,5 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-graphql</artifactId>
         </dependency>
-        <dependency>
```

Comment:
Shouldn't be the coverage converted to opentelemetry instead?

Reply:
I trust https://github.com/quarkus-qe/quarkus-test-suite/tree/main/http/graphql-telemetry if we need to extend coverage, it should be separate PR.